src/app/service/zappeditdb.service.ts

```diff
@@ -0,0 +1,21 @@
+
+import { Injectable } from '@angular/core';
+import Dexie, { Table } from 'dexie';
+import { User } from '../model/user';
+
+const DATASTORE = {
+  users: "++hexPubKey, name, displayName, pictureUrl, nip05, npub, about",
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ZappeditdbService extends Dexie{
+  ready = false;
+  users!: Table<User>;
```

Comment:
Since this is a follow list, can we call it "following" instead of "users"? in future, if we cache the users, then this will clash with the follow list.  Not sure if it will be easy to replace the client's stored cache with new one, in case we rename this key later.